test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -744,6 +748,26 @@
              (is (= 200 (:status @response)))
              (is (= hello-body (:body @response))))))))))
 
+(deftest double-stop-test
+  (testing "if the stop lifecycle is called more than once, we handle that gracefully and quietly"
+    (let [log-messages (atom [])]
+      (tk-log-testutils/with-logging-to-atom tk-log/root-logger-name log-messages
+        (let [app (tk-bootstrap/bootstrap-services-with-config
+                   [jetty9-service]
+                   {:webserver {:port 8080}})]
+          (tk-app/stop app)
```

Comment:
That's the thing, though, Jetty wasn't actually throwing an exception.  It was just logging about 100 of them and then swallowing them:

https://github.com/eclipse/jetty.project/blob/93a8d8e05f4120e57bc3c830d5c10e38dd4a9f97/jetty-jmx/src/main/java/org/eclipse/jetty/jmx/MBeanContainer.java#L272

So I really don't think there's anything I can test except logging.